cmake/dts.cmake

```diff
@@ -92,7 +92,7 @@ if(CONFIG_HAS_DTS)
     -include ${AUTOCONF_H}
     ${DTC_INCLUDE_FLAG_FOR_DTS}  # include the DTS source and overlays
     -I${ZEPHYR_BASE}/dts/common
-    -undef -D__DTS__
+    ${NOSYSDEF_CFLAG} -D__DTS__
```

Comment:
While you are patching this, could you move -D__DTS__ to the next line?

Right now it reads like "Don't define __DTS__".

Reply:
Changed in new revision.